nemo/collections/tts/modules/radtts.py

```diff
@@ -183,9 +182,7 @@ def __init__(
         self.speaker_embedding = torch.nn.Embedding(n_speakers, self.n_speaker_dim)
```

Comment:
Line 146 and 147: `n_hidden` and `mel_encoder_n_hidden` are not used anymore. Please remove these two params.
```
        n_hidden,
        mel_encoder_n_hidden,
```